test/appium/tests/medium/test_chats_m.py

```diff
@@ -996,3 +996,102 @@ def test_block_user_from_1_1_chat_header_check_mentions_and_push_notification_se
+        if not self.chat_1.chat_message_input.is_element_displayed():
+            self.drivers[0].fail('Chat input field has not appeared after contact request has been accepted')
+
+        self.home_1.just_fyi('Verify request sender can send messages to request acceptor after acceptance of cr')
+        self.chat_1.just_fyi("Clicking send button to clear input field from remained request message. Bug #13610")
+        self.chat_1.send_message_button.click()
+        message_from_sender = 'Message sent after my contact request has been accepted'
+        self.chat_1.send_message(message_from_sender)
+        if not chat_2.chat_element_by_text(message_from_sender).is_element_displayed():
+            self.errors.append('Message from request sender has not been received after acceptance of his request')
+
+        self.home_2.just_fyi('Verify contacts are mutually removed for users with enabled contact request')
+        chat_2.chat_options.click()
+        chat_2.view_profile_button.click()
+        chat_2.remove_from_contacts.click_until_absense_of_element(chat_2.remove_from_contacts)
+        chat_2.back_button.click()
+
+        chat_2.just_fyi('Verify cannot send messages to user who was removed from contacts')
+        if not chat_2.contact_request_button.is_element_displayed():
+            self.errors.append('Send contact request button is not displayed after removing user from contacts')
+        if chat_2.chat_message_input.is_element_displayed():
+            self.errors.append('Chat input field is displayed after removing user from contacts')
+
+        self.chat_1.just_fyi('Verify users are mutually removed from contacts')
+        if not self.chat_1.element_by_text('Not a contact').is_element_displayed():
+            self.errors.append('User has not been mutually removed from contacts of removed contact')
+        if not self.chat_1.contact_request_button.is_element_displayed():
+            self.errors.append('Send contact request button is not displayed after user has been removed from contacts')
+        if self.chat_1.chat_message_input.is_element_displayed():
+            self.errors.append('Chat input field is displayed after user has been removed from contacts')
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702377)
+    def test_mutual_cr_decline_contact_request(self):
+        [home.home_button.double_click() for home in (self.home_1, self.home_2)]
+        self.chat_1 = self.home_1.add_contact(self.public_key_2, add_in_contacts=False)
+        self.chat_1.send_contact_request('Contact request to decline')
+        self.home_2.handle_contact_request(self.default_username_1, accept=False)
+        self.home_2.home_button.click()
+        chat_2 = self.home_2.add_contact(self.public_key_1, add_in_contacts=False)
+
+        self.chat_1.just_fyi('Verify cannot sent messages to user who declined contact request')
+        if self.chat_1.chat_message_input.is_element_displayed():
```

Comment:
Typo, `Verify cannot send messages to user who declined contact request`

Reply:
Thanks @churik , fixed